students/smitco/lesson03/mailroom.py

```diff
@@ -20,7 +20,8 @@ def menu():
             card(name, value)
             flag = True
         elif choice == "2":
-            flag = False
+            report()
+            flag = True
         else:
             flag = False
```

Comment:
the `intro` function has some duplication, can you spot it and refactor?

Reply:
One of the tasks I struggled with for this exercise was to have the program return to the menu when a choice is completed. What I think I'm gathering now is that it would be better to have a simple menu function and then put the code to loop back to the menu in the name == main block instead of the menu function. Is that right?

For the intro function, I struggled to have it loop back to the original prompt after the user asks to see a list, so I did it the undesirable way of just copying the code block. Should I have used a while loop to get back to the input for the donor name?